projects/bounce/index.js

```diff
@@ -0,0 +1,48 @@
+const { sumTokensAndLPsSharedOwners } = require('../helper/unwrapLPs')
+
+const AUCTION = '0xA9B1Eb5908CfC3cdf91F9B8B3a74108598009096'
+const AUCTION_ETH_SLP = '0x0f8086d08a69ebd8e3a130a87a3b6a260723976f'
+
+const STAKING_ADDRESS = '0x98945BC69A554F8b129b09aC8AfDc2cc2431c48E'
+const STAKING_LP_ADDRESS = '0xbe5a88b573290e548759520a083a61051b258451'
+const MULTI_SIG_ADDRESS = '0xc9297466C6c7acc799Fb869806C53398b8B10680'
```

Comment:
this is your project treasury right? why are you counting it as tvl?

I have refactored the code here: https://github.com/DefiLlama/DefiLlama-Adapters/pull/7109/files